packages/salesforcedx-vscode-apex-debugger/test/unit/telemetry/telemetry.test.ts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { assert, SinonStub, stub } from 'sinon';
+import TelemetryReporter from 'vscode-extension-telemetry';
+import { TelemetryService } from '../../../src/telemetry/telemetry';
+
+describe('Telemetry', () => {
+  let reporter: TelemetryReporter;
+  let sendEvent: SinonStub;
+
+  beforeEach(() => {
+    reporter = new TelemetryReporter('salesforcedx-vscode', 'v1', 'test567890');
+    sendEvent = stub(reporter, 'sendTelemetryEvent');
+  });
+
+  afterEach(() => {
+    sendEvent.restore();
+    reporter.dispose();
+  });
+
+  it('Should send telemetry data', async () => {
+    const telemetryService = TelemetryService.getInstance();
+    telemetryService.initializeService(reporter, true);
+
+    telemetryService.sendExtensionActivationEvent();
+    assert.calledOnce(sendEvent);
+  });
+
+  it('Should not send telemetry data', async () => {
+    const telemetryService = TelemetryService.getInstance();
+    telemetryService.initializeService(reporter, false);
+
+    telemetryService.sendCommandEvent('debugger_command');
+    assert.notCalled(sendEvent);
+  });
+
+  it('Check telemetry sendCommandEvent data format', async () => {
```

Comment:
Nitpick. Is there a reason why we no longer use the word `Should` in the assert like we do in the previous two assertions?